client/layout/global-sidebar/style.scss

```diff
@@ -504,3 +514,32 @@ body.has-global-sidebar .help-center .components-card.help-center__container.is-
 body.has-global-sidebar.has-global-sidebar-collapsed .help-center .components-card.help-center__container.is-desktop {
 	left: 82px;
 }
+
+body.has-global-sidebar .is-global-sidebar-visible.is-support-session {
```

Comment:
Ideally, it'd be nice to load this CSS only when in a support session. (I'm aware there are other `.is-support-session` code)

Reply:
Since support session is a state rather than a component, I'm not sure where to put it except in the language switcher? This file also makes sense to me since we are changing the global sidebar UI 🙂

Reply:
I just wanted to share an idea 🙂 since there are not many codes for support sessions yet.

However, I have a concern about potential performance drawbacks. Loading "CSS in a different state" can often negatively impact load times, and isolating these issues after it is scaled can be challenging.

> This file also makes sense to me since we are changing the global sidebar UI

Yes, I think it makes sense to include the CSS in this file. 
It's not something that needs to be resolved immediately, but Calypso should have a mechanism to load different states of CSS for different critical paths. I've been thinking this consideration might also apply to the CSS and JS files for the Nav Redesign/Untangling but I did not jump into it.

Reply:
Could you share an article about this?